packages/api/db/migration/20220613141913_create_sensors_tables.js

```diff
@@ -0,0 +1,31 @@
+exports.up = function (knex) {
+  return Promise.all([
+    knex.schema.createTable('sensors', function (table) {
```

Comment:
@AbdusShaikh I should we merge your PR first, since we talked about how we wanted the table names to be singuler?

Reply:
I'd avoid merging the migrations at this point since changes are still being made here and there. My part is not yet ready so I cannot merge it first.